runcoms/README.md

```diff
@@ -10,61 +10,199 @@ in */etc* by default.
+|                  | `/etc/zshrc`                            |                                   | system provided startup script, for interactive shells, (local shells that live in an `XTerm`, `URxvt`, `gnome-terminal`) or subshells (like running `xterm` from the command prompt of a _login xterm_, | site login accounting, security monitors, site command logging policy
+|                  |                                         | `${ZDOTDIR:-$HOME}/.zshrc`        | user customization of their *interactive* environment for terminals                                                                                                                                      | sourcing custom completions (`gcloud.comp.inc`), custom user frameworks (zprezto is invoked here), additional path required by custom software installed in `/opt/*`
+|                  | `/etc/zshlogin`                         |                                   | additional customization for interactive **login** shells, such as those accessed via ssh                                                                                                                | tmux attach to existing session
+|                  |                                         | `${ZDOTDIR:-$HOME}/.zshlogin`     | user script for *login shell startup* (this shell is a session leader, and allocates a `ptty`                                                                                                            | aliases, keybindings, personal startup programs, time tracking apps, session managers, quote of the day, fortunes
+| ..               | ... your shell session happens here ... |
+|                  |                                         | `${ZDOTDIR:-$HOME}/.zshlogout`    | personal cleanup tasks                                                                                                                                                                                   | setting personal `crontabs` or `at` tasks, logging checout to your time tracking software, fortune to say goodbye.
+| `/etc/zshlogout` |                                         |                                   | site cleanup tasks                                                                                                                                                                                       | site command logging policy stop, login accounting records, security context spindown, etcThe configuration files are read in the following order:
+
+Discussion
+----------
+
+There are several places to add personal configuration and preferences in 
+the files already provided for startup, it’s just a matter of placing them  
```

Comment:
It's probably github's markdown parser's fault, but it looks like there's a manual newline after this in the rendered text.